pkg/cmd/cli/restic/server.go

```diff
@@ -92,6 +94,11 @@ func newResticServer(logger logrus.FieldLogger, baseName string) (*resticServer,
 		return nil, errors.WithStack(err)
 	}
 
+	err = validatePodVolumesHostPath(kubeClient, "/host_pods/")
```

Comment:
Ah I used a parameter here so that I could change the directory in the test. Another way to do it would be to use a var in the package and change it in the test. Is there any other way I can change the value in the test without using a var?

Reply:
Hm, I see. One way to address this is to have a struct that would contain both elements. That would be preferable to passing a hard-coded string imo.

Reply:
But now that I said that it sounds weird... 🤷‍♀

Reply:
another option: use a fake for the file system so you don't have to change the path -- see https://github.com/heptio/velero/blob/master/pkg/util/test/fake_file_system.go

Reply:
@skriss oh great, I was looking for something like that! That's a much better approach